### PR TITLE
Add project for astropy Table performance improvements

### PR DESCRIPTION
**Pull request by taldcroft**
@mhvk - if you have time it would be great to have you on-board as secondary mentor.  Understood that you have travel and other commitments.

### CODE REVIEW

File: _projects/2018/astropy/table-optimize.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+#### Milestones (if any)
+
+##### GSOC 2018 CODING STARTS
+
+* Perform developer install of `astropy`.
+* Install Python profiling tools.
+* Study astropy and the table package capabilities.
+
+##### GSOC 2017 MIDTERM
```
Comment (bsipocz) [inline _projects/2018/astropy/table-optimize.md]: should be 2018, it was a mistake in the template.

**Comment by bsipocz** [comment]
@taldcroft - Thanks! I only wonder of the timing of this project. Wouldn't most of the performace improvements be done by the summer for the 3.1 release?

**Comment by mhvk** [comment]
@taldcroft - in principle, not uninterested, but my main issue is that I'll be essentially unavailable from end of July to end of August, so I think I should not be formally listed.

File: _projects/2018/astropy/table-optimize.md (hunk 1 above)
Comment (taldcroft) [reply _projects/2018/astropy/table-optimize.md]: I knew about this, but then I tried to rush this before going out the door.

**Comment by taldcroft** [comment]
@bsipocz - about timing, there should be an email going out in the next couple of days to propose making releases happen in Mar / Sep, starting with 3.1 in September.  One immediate goal was to improve quality of life over the holidays for our release manager.  :smile:

**Comment by taldcroft** [comment]
@mhvk - OK, so you can participate on an as-available basis.

**Comment by mhvk** [comment]
Yes, that sounds good; I do think it is a good idea!  

And :+1: to releases at other times than June/December!

**Comment by taldcroft** [comment]
@bsipocz - I fixed the year.

**Comment by bsipocz** [comment]
The proposed new release dates are also super useful for GSoC, it was quite unfortunate that all those new features of the summer had to wait till the winter to for being in a release. So it's very much 💯 from me (and also like the astro connection to the equinoxes).

**Comment by bsipocz** [comment]
I'm adding this now, please open follow-up PRs when adjustments are needed (e.g it would be great to have a second mentor listed even if we plan to adjust it during student selection).

**Pull request merged**